### DISCUSSION TITLE
Remote Control of Playback over Bluetooth

### DISCUSSION THREAD

**Discussion by rjapenga** [bluefireteam/audioplayers#1757] [Q&A]
I cannot find anything in the documentation if the player supports bluetooth remote control (Play, Pause, Stop, Next, etc). The only thing I found was that the player was not working on bluetooth devices over a year ago. Does it support remote control? If not, are there plans for it? 

Thank you.

**Comment by Gustl22** [comment]
No, we have not implemented this. Also it's not the main scope of this library. If one is enthusiastic, we can think about a another package. But I personally am on my limit of spare time to spend :/
